Header borader
float64[] path_x
float64[] path_y
